packages/manager/src/features/LoadBalancers/LoadBalancerCreate/LoadBalancerLabel.tsx

```diff
@@ -23,6 +24,9 @@ export const LoadBalancerLabel = () => {
       data-qa-label-header
     >
       <TextField
+        onBlur={(e) => {
+          setFieldTouched(e.target.name, true); // mark the field as touched
+        }}
```

Comment:
I think `handleBlur` will work here.

```suggestion
    onBlur={handleBlur}
```